### DISCUSSION TITLE
be friendly to possibly correct requests

### DISCUSSION THREAD

**Discussion by harvey637** [emsesp/EMS-ESP32#1294] [Ideas]
Hi,
the iobroker adapter for ems-esp requested until v 2.4.0 for "custom" entities,
I guess, that the resposnse to that request somehow changed with the "new" 3.7.0-dev0,
at least I did not spot the logging of this error before.
In the ems-esp-iobroker-adapter this was remedied with introducing v2.4.1 with the switch to turn on/off the call for custom entities.
https://github.com/tp1de/ioBroker.ems-esp/issues/34

May be it is a better way to answer the api call to not existing entities (like custom entities, analogue sensors, ...) with "INFO" and the answer "nothing configured for <api-call customs|sensor|analogue|...>" instead of errors.

just my idea - no problem today, as v2.4.1 has this configurable.

Thank You+++
Harvey

**Comment by proddy** [comment]
So the change here is that when you have no Custom Entities configured, a call to `http://<hostname>/api/custom/` should return an empty JSON {} and not log an Error with the message "unknown device".

**Comment by harvey637** [comment]
Hi Proddy,Yes, petfect solution.Have a good Weekend!Ciao Harvey--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 17.09.23, 09:46 schrieb Proddy ***@***.***>:

   
  So the change here is that when you have no Custom Entities configured, a call to http://<hostname>/api/custom/ should return an empty JSON {} and not log an Error with the message "unknown device".
